lib/Interpreter/CppInterOp.cpp

```diff
@@ -1453,6 +1456,10 @@ namespace Cpp {
     }
   }
 
+  QualType GetBuiltinType(const std::string& name, Sema& sema) {
```

Comment:
This one seems unused and not part of the header. Maybe an oversight from the bigger PR?